src/views/Timeline.vue

```diff
@@ -271,6 +282,24 @@ export default {
     })
   },
   methods: {
+    setConnectionStatusNotification () {
+      switch (this.peerConnectionStatus) {
+        case 'PEER_CONNECTING':
+          this.connectionBarText = 'UI App is connecting to Ambianic Edge Device'
+          this.connectionBarVisibility = true
+          break
+        case 'PEER_CONNECTED':
+          this.connectionBarText = 'UI App has connected to an Ambianic Edge Device'
```

Comment:
@vickywane how about a shorter version:
"Connected to Ambianic Edge device"   ?